### ISSUE TITLE
Adding Custom http request headers

### ISSUE THREAD

**Issue by rd-devv** [megahertz/electron-simple-updater#61]
Hi megahertz, it's really a great module for auto-updates thank you for that.
The situation currently I have is that I am using AWS S3 bucket as a storage for my releases. I want to keep this storage private. I am able to do that with the help of electron-simple-publish package. The issue I have is that now while auto-update, to download these updates since these are private, is there any way that I can still download it with modifying the request headers or anything like that.

**Comment by megahertz** [comment]
Hi. Unfortunately, it's not possible. I can only manage http request to updates.json and to linux AppImage. All other files are downloaded by internal electron module, so I can't control these requests.

**Comment by rd-devv** [comment]
> Hi. Unfortunately, it's not possible. I can only manage http request to updates.json and to linux AppImage. All other files are downloaded by internal electron module, so I can't control these requests.

cool, thank you for quick reply will check if I can find any work around.

**Comment by rd-devv** [comment]
Hi megahertz, hope you're doing well. Actually as per my requirement following is the solution I have implemented.

I have created a server - which creates a pre-signedurl for the private file (.AppImage) from S3 bucket and given the path to the server in "feedUrl" for linux download.
This works fine for me on linux.

But for windows the squirrel required path to the directory and directory contains 3 files release, .nupkg, and .exe, so with the above approach how can accomplish this for windows. 

Any Ideas...
